### PR TITLE
#214: fixed message showing on block and menu add.

### PR DESCRIPTION
**Pull request by Mspiro**
**Testing Steps:**

- Add any new menu and see message appear on top
- Add any new block and see message appear on top
- Refer Below Screenshots

image

### CODE REVIEW

**Review by omkar-pd — APPROVED** [review]
Working fine !

image

File: src/Controllers/CustomBlock.php [hunk 1]
```diff
@@ -41,17 +44,19 @@ public function insertCustomBlock($twig){
         $variables['authenticated_userId'] = $_SESSION['user_id'];
         $variables['title'] = $this->reverie . " | Block";
         $baseUrl = $variables['base_url'];
-        header("Location:".$baseUrl."block-form");
-        echo $twig->render('block.html.twig');
+        $variables['status'] = "true";
```
Comment (vbadkar) [inline src/Controllers/CustomBlock.php:47]: Does the status message not appear beacuse ofthis redirection code?

**Review by vbadkar — CHANGES_REQUESTED** [review]
The messages are getting displayed but the redirection is not working now, Can you please look into this?

**Review by omkar-pd — APPROVED** [review]
Synced up with master.

image

**Comment by vbadkar** [comment]
Closing this pull request, as the purpose of this PR is not being served because if we pass the display message variable, we have to remove the redirection code as it redirects before passing the message variable, Hence no display message is displayed.

**Pull request closed without merging**